readme.rst

```diff
@@ -5,23 +5,44 @@ See the `Sphinx and Read the Docs <https://canonical-documentation-with-sphinx-a
 
 Then go through the following sections to use this starter pack to set up your docs repository.
 
-Download and install
---------------------
+Set up your documentation repository
+------------------------------------
 
-* *Incorporate this starter pack into an existing code repository* - copy all
-  the files from this repository into your project's directory structure,
-  and rename this directory to ``docs`` or similar.
+You can either create a standalone documentation project based on this repository or include the files from this repository in a dedicated documentation folder in your code repository.
 
-* *Start a standalone documentation project* - clone this locally and start
-  working.
+**Note:** We're planning to provide the contents of this repository as an installable package in the future, but currently, you need to copy and update the required files manually.
 
-In documentation directory, run::
+Standalone documentation repository
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To create a standalone documentation repository, clone this starter pack repository, `update the configuration <#configure-the-documentation>`_, and then commit all files to your own documentation repository.
+
+You don't need to move any files, and you don't need to do any special configuration on Read the Docs.
+
+Documentation in the code repository
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To add your documentation to your code repository, create a dedicated documentation folder in your repository (for example, ``docs``).
+
+Copy the files from this starter pack repository into this ``docs`` folder.
+Then do the following changes:
+
+- Move the workflow from the ``docs/.github`` folder into the ``.github`` folder of the root directory of your repository, or include the jobs from the workflow file into the workflows you have already set up for your code repository.
```

Comment:
"Move the workflow files from the [...] of your code repository, or include the job logic from the files into your existing workflows."

By the way, file `automatic-check.yaml` should be renamed to something like `automatic-doc-checks.yaml` since one option above is to move the file into an existing repository.